client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -171,8 +171,8 @@ export default React.createClass( {
 						<Button compact primary onClick={ this.props.updateAllPlugins } >
 							{
 								this.translate(
-									'Update %(numUpdates)d Plugin',
-									'Update %(numUpdates)d Plugins',
+									'Updated %(numUpdates)d Plugin',
```

Comment:
Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)